### PR TITLE
[FSDE-15440] Updated portal theme Stat component design and docs

### PR DESCRIPTION
**Pull request by mattnickles**
https://ctl.atlassian.net/browse/FSDE-15440

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-333/1/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-333/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://e3cb8262c42c7e86a589e6b7d0c41796ae18e722-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-333-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-333/3/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-333/4/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://22db2b0c7a7b12ade37e644f3d679b8d6838fefd-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-333-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by janibolkvadze — APPROVED** [review]

File: src/chi/components/stat/stat.scss [hunk 1]
```diff
@@ -111,6 +111,10 @@
           left: $stat-background-icon-icon-left;
           position: relative;
           width: 4.5rem;
+
+          &.outline {
```
Comment (janibolkvadze) [inline src/chi/components/stat/stat.scss:115]: Since this is a complimentary class, do we need a `-` prefix here?
@mattnickles @jllr

Comment (mattnickles) [reply src/chi/components/stat/stat.scss:115]: @janibolkvadze This is targeting an existing class in the marketing icon web component which does not have a `-` prefix. Agree it makes sense to rename but I would recommend doing that in a separate PR.

Comment (janibolkvadze) [reply src/chi/components/stat/stat.scss:115]: Sounds good

**Pull request merged**